library/std/src/keyword_docs.rs

```diff
@@ -1064,7 +1064,7 @@ mod move_keyword {}
 /// ```rust,compile_fail,E0502
 /// let mut v = vec![0, 1];
 /// let mut_ref_v = &mut v;
-/// ##[allow(unused)]
+/// # #[allow(unused)]
```

Comment:
What is this `## -> # #`?

Reply:
This is a workaround for https://github.com/rust-lang/rust/issues/136899. Sorry, should have mentioned that in the description.